src/libs/pool-data-provider/hooks/use-incentives-data-context.tsx

```diff
@@ -36,6 +36,7 @@ import {
 export interface IncentivesContext {
   reserveIncentives: ReserveIncentiveDict;
   userIncentives: UserIncentiveDict;
+  usersIncentives: UserIncentiveDict[];
```

Comment:
why `users`? the data is still for a single user

Reply:
resolved